applications/nrf_desktop/README.rst

```diff
@@ -2000,18 +2012,30 @@ Background Device Firmware Upgrade
 ==================================
 
 The nRF Desktop application uses the :ref:`nrf_desktop_config_channel` and :ref:`nrf_desktop_dfu` for the background DFU process.
-From the application perspective, the update image transfer during the background DFU process is very similar for both MCUboot and B0 bootloader.
+From the application perspective, the update image transfer during the background DFU process is very similar for all supported configurations:
+
+* MCUboot
+* Secure Bootloader (B0)
```

Comment:
Immutable bootloader?

Reply:
It is listed here as it has been referred in this docs already in line 1886, so I would keep it as is:

```
**Secure Bootloader**
  In this documentation, the Secure Bootloader is referred as *B0*.
```